app/Models/Category.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use App\Models\Product;
+use App\Models\Suggest;
+
+class Category extends Model
+{
+    protected $table = 'categories';
+    protected $fillable = [
+        'category_name',
+    ];
+
+    public function product()
```

Comment:
```suggestion
    public function products()
```
hasMany thì đặt tên function là số nhiều, hasOne với belongTo thì đặt tên số ít e nhé